packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md

```diff
@@ -214,15 +214,15 @@ Okta uses `count=100` as the pagination reference to return up to 100 elements.
 
 The SCIM server must consistently return the same ordering of results for the requests, regardless of which values are provided for the `count` and `startIndex` pagination references. For more information on pagination, see [Section 3.2.2.3](http://www.simplecloud.info/specs/draft-scim-api-01.html#query-resources) of the V1.1 specification.
 
-A sample request from Okta to retrieve the users from the SCIM app:
+A sample request from Okta to retrieve the users from the SCIM application:
 
 ```http
 GET /scim/v1/Users?startIndex=1&count=100 HTTP/1.1
 User-Agent: Okta SCIM Client 1.0.0
 Authorization: <Authorization credentials>
 ```
 
-The response to this request is a JSON listing of all the resources found in the SCIM app.
+The response to this request is a JSON listing of all the resources found in the SCIM application.
 
 ### Retrieving a specific user
```

Comment:
Retrieve a specific User

Reply:
Capital U in user?

Reply:
Fixed here and elsewhere where User and user have separate meanings